### ISSUE TITLE
[FEATURE]:  remotebuzzer on live chroma 

### ISSUE THREAD

**Issue by andi34** [PhotoboothProject/photobooth#53]
### Is your feature request related to a problem?

No

### Description

Remotebuzzer Server is actually disabled on livechroma.

### Describe the solution you'd like

Enable support for livechroma.

### Describe alternatives you've considered

None.

**Comment by andi34** [comment]
Keep open

**Comment by andi34** [comment]
Still todo

**Comment by flighter18** [comment]
@andi34 who is the progress of this feature? 
maybe I can help you?
I would love to take pictures in chroma by buzzer.

**Comment by andi34** [comment]
@flighter18 might be it works already, can't say as I haven't used the keying for a long time now 😅
Restructuring and rewriting other sources had and have priority in the limited time I am able to spend to the project currently. Having 3 child's now which makes development more difficult to me.

**Comment by flighter18** [comment]
@andi34 , thanks for the response!
I tested the buzzer in keying mode, but it doesn't seem to work.
It looks like the remote buzzer server gets loaded when the page loads, but when I use curl with the server URL and "commands/start-picture," nothing happens.
For easier testing, I changed the api.takePicture function in "remotebuzzer-client.js" to photoBooth.thrill('chroma'), so it should take a chroma picture.

loading the page, remote buzzer gets startet:
<img width="620" alt="Bildschirmfoto 2024-12-29 um 22 16 19" src="https://github.com/user-attachments/assets/ce25c7af-3092-447b-98ec-576a90ec6ecd" />


executing the curl command:
<img width="451" alt="Bildschirmfoto 2024-12-29 um 22 16 25" src="https://github.com/user-attachments/assets/8184ba0b-a410-4fcf-b079-14b53cd8e30a" />


changed remotebuzzer-client.js:
<img width="410" alt="Bildschirmfoto 2024-12-29 um 22 16 37" src="https://github.com/user-attachments/assets/3410e147-230f-4cc3-86ae-3f050c309656" />

any hint? :)

**Comment by andi34** [comment]
Right, it needs to trigger `photoBooth.thrill('photo');`. It's working while pressing the capture button at the point where trying the curl?
Anything visible inside the Remotebuzzer log?
